### PR TITLE
CLOUDP 171401 Add crud commands "atlas dataLakePipeline runs"

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Add crud commands "atlas dataLakePipeline runs"

_Jira ticket:_ CLOUDP-171401

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/datalakepipelines/runs/describe.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	}
+
+	cmd.Flags().StringVar(&opts.pipelineName, flag.Pipeline, "", usage.Pipeline)
+	_ = cmd.MarkFlagRequired(flag.Pipeline)
+
+	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
+	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
+	_ = cmd.RegisterFlagCompletionFunc(flag.Output, opts.AutoCompleteOutputFlag())
+
+	_ = cmd.MarkFlagRequired(flag.ClusterName)
```
Comment (gssbzn) [inline internal/cli/atlas/datalakepipelines/runs/describe.go:96]: 👀 I don't see this flag

File: internal/cli/atlas/datalakepipelines/runs/describe_test.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	"github.com/stretchr/testify/assert"
+	atlasv2 "go.mongodb.org/atlas/mongodbatlasv2"
+)
+
+func TestDescribe_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockPipelineRunsDescriber(ctrl)
+
+	expected := &atlasv2.IngestionPipelineRun{
+		Id:          pointer.Get[string]("507f1f77bcf86cd799439011"),
```
Comment (gssbzn) [inline internal/cli/atlas/datalakepipelines/runs/describe_test.go:40]: [q] do you need the type hint? usually it should be able to autodiscover it

File: internal/cli/atlas/datalakepipelines/datalakepipelines.go
Comment (gssbzn) [inline internal/cli/atlas/datalakepipelines/datalakepipelines.go:1]: shouldn't the name of this file be data_lake_pipelines.go?

Comment (fmenezes) [reply internal/cli/atlas/datalakepipelines/datalakepipelines.go:1]: thanks for this comment, I finally understand the rule files use '_' folders/packages don't

**Review by gssbzn — APPROVED** [review]
:shipit:

**Review by kanchana-mongodb — APPROVED** [review]
LGTM

**Pull request merged**